### DISCUSSION TITLE
Custom ordering of findings

### DISCUSSION THREAD

**Discussion by aronmolnar** [Syslifters/sysreptor#13] [General]
# Problem
Currently, the CVSS vector is a mandatory field for findings. Findings are ordered by CVSS score during reporting (and in the findings array in the HTML report).

# Solution
It should be possible to customize the ordering and get rid of the mandatory CVSS score. It should be possible to sort by multiple custom-defined fields (e.g. first per target, second by severity).

It might also be desirable to allow custom ordering by drag&drop in the findings section.

# Drawback
None

**Comment by Lednerb** [comment]
This is definitely my top wanted and needed feature for this awesome tool!

**Comment by MWedl** [comment]
We are starting to implement this feature. See #69

**Comment by aronmolnar** [comment]
Implemented in the latest version
